src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx

```diff
@@ -16,80 +16,83 @@ import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application
-    applicationStore.setIsLoading(true);
-    const signatureHash = await walletStore.web3.eth.personal.sign(userInput, walletStore.accountAddress);
-    setErrorMessage('');
-    Operator.getInstance().updateOperatorName(operator.id, signatureHash, userInput).then((response) => {
-      operator.name = response;
-      applicationStore.setIsLoading(false);
-      const selectedOperator = myAccountStore.ownerAddressOperators.find((op: any) => op.id === operator.id);
-      selectedOperator.name = response;
-      navigate(-1);
-    }).catch((error: any) => {
-      console.log('<<<<<<<<<<<error>>>>>>>>>>>');
-      console.log(error.response.data);
-      setErrorMessage(error.response.data.error.message);
-      console.log('<<<<<<<<<<<error>>>>>>>>>>>');
-      applicationStore.setIsLoading(false);
-    });
-  };
+    const submitOperatorName = async () => {
+        try {
+            applicationStore.setIsLoading(true);
+            const signatureHash = await walletStore.web3.eth.personal.sign(userInput, walletStore.accountAddress);
+            setErrorMessage('');
+            Operator.getInstance().updateOperatorName(operator.id, signatureHash, userInput).then((response) => {
+                operator.name = response;
+                applicationStore.setIsLoading(false);
+                const selectedOperator = myAccountStore.ownerAddressOperators.find((op: any) => op.id === operator.id);
+                selectedOperator.name = response;
+                console.log(response);
+                navigate(-1);
+            });
+        } catch (error: any) {
+            console.log('<<<<<<<<<<<error>>>>>>>>>>>');
+            setErrorMessage(error.message);
+            console.log('<<<<<<<<<<<error>>>>>>>>>>>');
+            applicationStore.setIsLoading(false);
+        }
+    };
 
-  const setOperatorName = async (e: any) => {
-    const textInput = e.target.value.trim();
-    setUserInput(textInput);
-  };
+    const setOperatorName = async (e: any) => {
+        const textInput = e.target.value.trim();
+        setUserInput(textInput);
+    };
 
-  const submitDisable = userInput.length === 0;
+    const submitDisable = userInput.length === 0;
 
-  return (
-      <BorderScreen
-          blackHeader
-          header={'Change Operator Name'}
-          body={[
-            (
-                <Grid container item>
-                  <Grid container style={{ gap: 24 }}>
-                    <Grid item className={classes.Text}>
-                      {/*Enter an Ethereum address that will receive all of your validators block proposal rewards. <LinkText text={'What are proposal rewards?'} link={'http://google.com'} />*/}
+    return (
+        <BorderScreen
+            blackHeader
+            header={'Change Operator Name'}
+            body={[
+                (
+                    <Grid container item>
+                        <Grid container style={{ gap: 24 }}>
+                            <Grid item className={classes.Text}>
```

Comment:
Keep one variant of code styling. After `{{` and before `}}` empty spaces are absent in most of the cases, keep it the same

Reply:
done